### PR TITLE
Add Chained Builds Announcement to OCP 3.7 Release Notes

### PR DESCRIPTION
**Pull request by bmcelvee**

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL ❄️

File: release_notes/ocp_3_7_release_notes.adoc [hunk 1]
```diff
@@ -1195,6 +1195,14 @@ perform new tasks.
...
 CLI] for information on how to install and write extensions for the CLI.
 
+[[ocp-37-chained-builds]]
+==== Chained Builds
+
+In {product-title} 3.7,
+xref:../dev_guide/builds/advanced_build_operations.html#dev-guide-chaining-builds[Chaining
+Builds] is a better approach for producing runtime-only
+application images, and fully replaces the Extended Builds feature.
+
```
Comment (mburke5678) [inline release_notes/ocp_3_7_release_notes.adoc]: Do users know what chain builds are?

**Comment by mburke5678** [comment]
@bmcelvee One question

File: release_notes/ocp_3_7_release_notes.adoc (hunk 1 above)
Comment (bmcelvee) [reply release_notes/ocp_3_7_release_notes.adoc]: Good point, @mburke5678! Chained Builds have been in the documentation for a while, but I went ahead and included the benefits listed in the original announcement. Do you think that is sufficient?

**Comment by mburke5678** [comment]
@bmcelvee LGTM

**Comment by bmcelvee** [comment]
@bparees PTAL ❄️

**Comment by bparees** [comment]
@bmcelvee looks great.  are we crossing our fingers for snow? :)

**Comment by bmcelvee** [comment]
@bparees, thanks! Usually, but not enough to be snowed in. :)

**Pull request merged**